src/layouts/MainLayout.astro

```diff
@@ -15,11 +15,11 @@ const isFallback = !!Astro.params.fallback;
 const url = new URL(Astro.request.url);
 const currentPage = url.pathname;
 const currentFile = `src/pages${currentPage.replace(/\/$/, '')}.md`;
-const githubEditUrl = `https://github.com/withastro/docs/blob/main/${currentFile}`;
 const t = useTranslations(Astro);
 const formatTitle = (content) => (content.title ? `${content.title} 🚀 ${t('site.title')}` : t('site.title'));
 const lang = getLanguageFromURL(url.pathname);
 const bcpLang = normalizeLangTag(lang);
+const githubEditUrl = content.githubURL && lang === 'en' ? `${content.githubURL}README.md` : `https://github.com/withastro/docs/blob/main/${currentFile}`;
```

Comment:
Thanks for taking this on @Yan-Thomas!

What do you think about doing this? Currently the edit link on non-English pages that use the fallback content results in a 404 because the translated version doesn’t exist yet. This change would link to the source README also for fallback pages.

```suggestion
const githubEditUrl = content.githubURL && (lang === 'en' || isFallback)
  ? `${content.githubURL}README.md`
  : `https://github.com/withastro/docs/blob/main/${currentFile}`;
```

Reply:
Nice catch, thanks! 🙌

Reply:
I think the issue still persists, I'll take a look at this problem 🤔

Reply:
Now I think the issue is solved (I hope at least 😅)